ts/Core/Axis/Axis.ts

```diff
@@ -271,49 +404,93 @@ class Axis {
         userMax?: number;
         userMin?: number;
     };
+    /** @internal */
     public opposite?: boolean;
+    /** @internal */
     public options!: (AxisOptions|XAxisOptions|YAxisOptions);
+    /** @internal */
     public ordinal?: AxisComposition['ordinal'];
+    /** @internal */
     public overlap!: boolean;
+    /** @internal */
     public paddedTicks!: Array<number>;
+    /** @internal */
     public plotLinesAndBands!: Array<PlotLineOrBand>;
+    /** @internal */
     public plotLinesAndBandsGroups!: Record<string, SVGElement>;
+    /** @internal */
     public pointRange!: number;
+    /** @internal */
     public pointRangePadding!: number;
+    /** @internal */
     public pos!: number;
+    /** @internal */
     public positiveValuesOnly!: boolean;
+    /** @internal */
     public reserveSpaceDefault?: boolean;
+    /** @internal */
     public reversed?: boolean;
+    /** @internal */
     public right!: number;
+    /** @internal */
     public sector?: number;
+    /** @internal */
     public series!: Array<Series>;
+    /** @internal */
     public showAxis?: boolean;
+    /** @internal */
     public side!: number;
+    /** @internal */
     public single?: boolean;
+    /** @internal */
     public softThreshold?: boolean;
+    /** @internal */
     public staggerLines?: number;
+    /** @internal */
     public staticScale?: number;
+    /** @internal */
     public threshold?: number;
+    /** @internal */
     public thresholdAlignment?: number;
+    /** @internal */
     public tickAmount!: number;
+    /** @internal */
     public tickInterval!: number;
+    /** @internal */
     public tickmarkOffset!: number;
+    /** @internal */
```

Comment:
https://api.highcharts.com/class-reference/Highcharts.Axis#tickPositions not internal